Makefile

```diff
@@ -32,37 +37,45 @@ help:
 	@echo "  eslint           Runs a code linter on the JavaScript code"
 	@echo "  dumpdb           Create a postgres database dump with timestamp used as file name"
 	@echo "  loaddb           Load a database dump into postgres, file name in DB_DUMP_FILE"
-	@echo "  build-frontend   Builds the frontend static files"
-	@echo "  build-frontend-w Watches the frontend static files and builds on change"
+	@echo "  build-tagadmin   Builds the tag_admin frontend static files"
+	@echo "  build-tagadmin-w Watches the tag_admin frontend static files and builds on change"
 	@echo "  sync-projects    Runs the synchronization task on all projects"
 	@echo "  requirements     Compiles all requirements files with pip-compile\n"
 
-.docker-build:
-	make build
-
-build:
-	cp ./docker/config/webapp.env.template ./docker/config/webapp.env
-	sed -i -e 's/#SITE_URL#/$(subst /,\/,${SITE_URL})/g' ./docker/config/webapp.env
+.frontend-build:
+	make build-frontend
+.webapp-build:
+	make build-webapp
 
+build: build-frontend build-webapp
+build-frontend: webapp-env
+	"${DC}" build frontend
+	touch .frontend-build
+build-webapp: webapp-env
 	"${DC}" build --build-arg USER_ID=$(USER_ID) --build-arg GROUP_ID=$(GROUP_ID) webapp
+	touch .webapp-build
 
-	touch .docker-build
+webapp-env:
+	cp ./docker/config/webapp.env.template ./docker/config/webapp.env
+	sed -i -e 's/#FRONTEND_URL#/$(subst /,\/,${FRONTEND_URL})/g;s/#SITE_URL#/$(subst /,\/,${SITE_URL})/g' ./docker/config/webapp.env
 
-setup: .docker-build
+setup: .webapp-build
 	"${DC}" run webapp //app/docker/set_up_webapp.sh
 
-run: .docker-build
-	"${DC}" run --rm --service-ports webapp
+run: .frontend-build .webapp-build
+	"${DC}" up --no-build frontend webapp
 
 clean:
-	rm .docker-build
+	rm -f .docker-build .frontend-build .webapp-build
```

Comment:
I guess you can remove `.docker-build` here, as it's not being used anymore.

Reply:
I left it in for now, to make it easier for anyone that has a current Pontoon checkout. If this gets cleaned up sometime later rather than now, anyone who's run `make clean` at least once in the interim won't need to manually remove the old empty target file.